invenio_communities/searchapp.py

```diff
@@ -17,14 +17,14 @@
 def search_app_context():
     """Search app context processor."""
     return {
-        #The Following functions should be defined for the communties
+        # The Following functions should be defined for the communties
```

Comment:
can we remove all these ones now?

Reply:
not everything is defined, so i am not sure.

Reply:
@Pineirin can you please help here?